sys/include/color.h

```diff
@@ -127,6 +127,45 @@ static inline void color_rgb_invert(const color_rgb_t *rgb, color_rgb_t *inv_rgb
     inv_rgb->b = rgb->b ^ 0xFF;
 }
 
+/**
+ * @brief Shifts a given rgb color to change it's brightness
+ *
+ * @pre                 ((rgb != NULL) && (out != NULL))
+ *
+ * @param[in] rgb       Input rgb color, that should be shifted. Must be NOT NULL
+ * @param[out] out      Output rgb color, result of the shift. Must be NOT NULL
+ * @param[in] shift     Amount by which the color components should be shifted.
+ *                      May be positive (shift to left) or negative (shift to right).
+ */
+static inline void color_rgb_shift(const color_rgb_t *rgb, color_rgb_t *out, int8_t shift)
```

Comment:
I'm personally a fan of having the first argument always being the destination. (Unless a file descriptor or handle or whatever is there, which would go even before.)

This would be consistent with the C lib or POSIX, e.g. `memcpy()`, `memmove()`, `strcpy()`, `strncpy()`, `recvmsg()`, `recvfrom()` etc. always have the destination as first argument (unless a fd/handle is present, than it is the second).

Reply:
While I agree with you, the rest of [`color.h`](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/color.h) does it the other way round.

Reply:
OK. Then consistency within color.h is preferable.